examples/counter/Cargo.toml

```
@@ -4,4 +4,4 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-kobold = { path = "../../crates/kobold" }
+kobold = { path = "../../crates/kobold", version = "0.5.0" }
```

Examples aren't being published on crates.io so they don't need this.

```suggestion
kobold = { path = "../../crates/kobold" }
```